AllReadyApp/Web-App/AllReady/Controllers/AccountController.cs

```diff
@@ -411,5 +411,29 @@ private static void RetrieveFirstAndLastNameFromExternalPrincipal(ExternalLoginI
             firstName = array[0];
             lastName = array[1];
         }
+        
+        //Area Names
+        private const string ADMIN_AREA = "Admin";
+
+        //Controller Names
+        private const string HOME_CONTROLLER = "Home";

```

Comment:
To add to this I'd read MS guidance in some of their GItHub issues not to rely on nameof for getting the controller or action names. There are attributes which can override the action name which could then differ from the method name, so nameof could return incorrect results.